about.html

```diff
@@ -30,32 +37,33 @@
                 </g>
               </svg>
             </a>
-            <div>
-                <ul class="flex nav text-dark">
-                    <li><a href="./about.html">About</a></li>
-                    <li><a href="#">Program</a></li>
-                    <li><a href="#">Join</a></li>
+            <div class="navbar flex">
+                <ul class="flex nav">
+                    <li><a class="link" href="./about.html">About</a></li>
```

Comment:
- [ ] [OPTIONAL] Kindly add a link to the home page from the about page, so the user can navigate easily between the pages

Reply:
The requirements for the project required the logo to be linked to the home page.